### PR TITLE
docs: clarify cloudflare env usage

### PR DESCRIPTION
**Pull request by RomanistHere**
I fund the description about env variables not clear and confusing 

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

**Comment by ghostdevv** [comment]
Perhaps instead of adding a note we should rename that section of the docs from `Environment Variables` to `Bindings`? Though I haven't tested what happens with `$env/dynamic` and the cloudflare adapter before

File: documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ export async function POST({ request, platform }) {
 }
 ```
 
+> It doesn't affect how [static variables](https://kit.svelte.dev/docs/modules#$env-static-private) work. They cant still be accessed as imports from `$env/static/private`.
```
Comment (ghostdevv) [inline documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md:77]:
```suggestion
> It doesn't affect how [static variables](https://kit.svelte.dev/docs/modules#$env-static-private) work. They can still be accessed as imports from `$env/static/private`.
```

Comment (benmccann) [inline documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md:77]: I don't think this is correct. I don't have a Cloudflare account, so haven't tried it, but from a glance at the code I believe all of Kit's environment variable modules are available. I.e. `$env/static/private`, `$env/dynamic/private`, `$env/static/public`, and `$env/dynamic/public`.

Even if it were only static environment variables that were still to be accessed in that manner I find it odd that you refer to only `$env/static/private`. Btw, the link should simply be `modules#$env-static-private`

Comment (RomanistHere) [reply documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md]: I haven't checked other three, but `$env/static/private` was available. I simply created GET endpoint to return variable and it was showing up in the response.

I'll see if I have time to create a repo to test if no one else can confirm it works this way.

Comment (RomanistHere) [reply documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md]: I haven't checked other three, but `$env/static/private` was available. I simply created GET endpoint to return variable and it was showing up in the response.

I'll see if I have time to create a repo to test if no one else can confirm it works this way.

File: documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md [hunk 2]
```diff
@@ -74,6 +74,8 @@ export async function POST({ request, platform }) {
 }
 ```
 
+> It doesn't affect how [static variables](https://kit.svelte.dev/docs/modules#$env-static-private) work. They can still be accessed as imports from `$env/static/private`.
```
Comment (benmccann) [inline documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md:77]:
```suggestion
> SvelteKit's built-in `$env` module should be preferred for environment variables.
```

File: documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md (hunk 1 above)
Comment (benmccann) [reply documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md]: I've renamed it to Bindings and updated the other text as well, so will go ahead and merge this

**Pull request merged**